dashboard/test/controllers/api/v1/peer_review_submissions_controller_test.rb

```diff
@@ -5,27 +5,37 @@ class Api::V1::PeerReviewSubmissionsControllerTest < ActionController::TestCase
 
   setup_all do
     @course_unit = create :plc_course_unit
-    @level_1 = create :free_response, peer_reviewable: true
-    @level_2 = create :free_response, peer_reviewable: true
-    @level_3 = create :free_response, peer_reviewable: true
+    @level_1, @level_2, @level_3, @level_4 = create_list :free_response, 4, peer_reviewable: true
 
-    [@level_1, @level_2, @level_3].each do |level|
+    [@level_1, @level_2, @level_3, @level_4].each do |level|
       create :script_level, script: @course_unit.script, levels: [level]
     end
 
     @submitter = create :teacher
     reviewer = create :teacher
 
-    [@level_1, @level_2, @level_3].each do |level|
+    [@level_1, @level_2, @level_3, @level_4].each do |level|
+      level_source = create :level_source, level: level
+      create :user_level, user: @submitter, level: level, level_source: level_source
       2.times do
-        create :peer_review, submitter: @submitter, script: @course_unit.script, level: level, data: nil
+        PeerReview.create(submitter: @submitter, script: @course_unit.script, level: level, data: nil, level_source_id: level_source.id)
```

Comment:
I'd love to still use the factory if we can; would that just be a matter of extending the definition to allow overriding `level_source_id`?